idb/postgres/internal/encoding/encode.go

```diff
@@ -60,6 +96,11 @@ func convertSignedTxnWithAD(stxn types.SignedTxnWithAD) types.SignedTxnWithAD {
 
 // EncodeSignedTxnWithAD returns a json string where all byte arrays are base64 encoded.
 func EncodeSignedTxnWithAD(stxn types.SignedTxnWithAD) []byte {
+	ap := stxn.Txn.AssetParams
+	ap.AssetName = strings.ReplaceAll(ap.AssetName, "\000", "")
+	ap.UnitName = strings.ReplaceAll(ap.UnitName, "\000", "")
+	ap.URL = strings.ReplaceAll(ap.URL, "\000", "")
```

Comment:
This looks incorrect, but I'm probably not understanding the context : 
going from *imported* AssetParams to persisted AssetParams should base64 the result into the "XXXBytes" variant only when needed. Otherwise, it should remain in the original variable.

Reply:
This part is for the transaction's JSON format, these fields happen to be unused but I left it in like this in case Anya wants to use it.